liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/PluginConfigSupport.java

```diff
@@ -56,14 +56,14 @@
     /**
      * Loose application.
      */
-    @Parameter(property = "looseApplication", defaultValue = "false")
+    @Parameter(property = "looseApplication", defaultValue = "true")
     protected boolean looseApplication;
 
     /**
      * Packages to install. One of "all", "dependencies" or "project".
      */
-    @Parameter(property = "installAppPackages", defaultValue = "dependencies")
-    private String installAppPackages;
+    @Parameter(property = "deployPackages", defaultValue = "dependencies", alias = "installAppPackages")
```

Comment:
Did we talk about having the default be `project` at some point?

Reply:
Not sure. It definitely seems like a more common use case than `dependencies`. I'm going to change it over to `project`.

Reply:
@chyt Do you agree with changing the default to `project`?